src/storage/file_manager.c

```diff
@@ -5357,6 +5357,7 @@ file_alloc (THREAD_ENTRY * thread_p, const VFID * vfid, FILE_INIT_PAGE_FUNC f_in
 	{
 	  /* commit and undo (to deallocate) */
 	  log_sysop_end_logical_undo (thread_p, RVFL_ALLOC, NULL, UNDO_DATA_SIZE, undo_log_data);
+	  logpb_force_flush_pages (thread_p);
```

Comment:
Is it necessary?

Reply:
No. Sorry, it was only used to reproduce the issue. I forgot to remove it.